### ISSUE TITLE
Two Merchants showed up today

### ISSUE THREAD

**Issue by Magleose** [TangoTek/TekTopia-Community#235]
**Describe the bug**
Not sure if this is working as intended or not, but I had 2 merchants show up today

**To Reproduce**
1. see attached screen shot
2. Not sure how it happened, so no instructions on how to reproduce
3. See error

**Expected behavior**
I was only expecting one merchant per day

**Screenshots**

![2019-05-31_18 58 50](https://user-images.githubusercontent.com/37353615/58739239-41673180-83d7-11e9-99d6-d7325c99a905.png)

** Server/Location **
Main(Survival) 2
1357 / 70 / 5261

**Additional context**

**Comment by JohnyDL** [comment]
When did it happen? near a restart or near when you were logging in?

**Comment by Magleose** [comment]
nope, it was about an hour after restart, and I had been playing all day

**Comment by JohnyDL** [comment]
You didn't walk/fly into your village at some point from a distance?

**Comment by ekkelos** [comment]
![2 mercents](https://user-images.githubusercontent.com/16520267/59267182-d2e16980-8c49-11e9-99fd-a07d25aa484a.jpg)
i had to merchents show up to my village

**Comment by TheGreatNaj** [comment]
![2019-06-23_01 17 28](https://user-images.githubusercontent.com/25763105/59972013-0333dc80-9555-11e9-8788-211485354fc7.png)
Same lol I think one was from the day before that didn't leave though. Im not sure but I think its the trades from the day before. It just didnt despawn like it was supposed to. Possibly because I was farther away from the town from dusk until day again?

**Comment by TheGreatNaj** [comment]
Strangely enough the following days after having the 2 merchants, no merchants will spawn.
